packages/server/src/fhircast/routes.test.ts

```diff
@@ -23,6 +23,7 @@ describe('FHIRCast routes', () => {
   beforeAll(async () => {
     const config = await loadTestConfig();
     await initApp(app, config);
+    await getRedis().flushdb();
```

Comment:
Out of curiosity, did you find that these calls to `flushdb()` were necessary, or more of a hygiene thing?

Reply:
Yes in some cases they were necessary